src/fondant/core/component_spec.py

```diff
@@ -318,10 +318,6 @@ def default_arguments(self) -> t.Dict[str, Argument]:
             ),
         }
 
-    @property
-    def kubeflow_specification(self) -> "KubeflowComponentSpec":
```

Comment:
I'm contemplating removing the `KubeflowComponentSpec` from the component_spec.py and integrating it more in the compiler (the only place where it is used)

Reply:
yes please